### PR TITLE
EPMRPP-80065 || Filter names fixes

### PR DESCRIPTION
**Pull request by tr1ble**

### CODE REVIEW

File: app/src/pages/inside/common/utils.js [hunk 1]
```diff
@@ -157,8 +157,9 @@ export const getGroupedDefectTypesOptions = (
     });
     defectTypesOptions = defectTypesOptions.concat(
       defectTypeGroup.map((defectType) => {
-        const label = defectTypesLocalization[defectType.typeRef.toLowerCase()]
-          ? formatMessage(defectTypesLocalization[defectType.typeRef.toLowerCase()])
+        const defectTypeLabelId = `Defect_Type_${defectType.locator.toUpperCase()}`;
+        const label = defectTypesLocalization[defectTypeLabelId]
+          ? formatMessage(defectTypesLocalization[defectTypeLabelId])
```
Comment (Vadim73i) [inline app/src/pages/inside/common/utils.js:162]: Why is this all, if you can just 
label: defectType.longName,

**Review by Vadim73i — CHANGES_REQUESTED** [review]

File: app/src/pages/inside/common/utils.js (hunk 1 above)
Comment (tr1ble) [reply app/src/pages/inside/common/utils.js:162]: It's done for default defect types translation

**Review by Vadim73i — APPROVED** [review]

File: app/src/pages/inside/common/utils.js [hunk 2]
```diff
@@ -169,6 +172,9 @@ export const getGroupedDefectTypesOptions = (
           color: defectType.color,
           locator: defectType.locator,
           meta: {
+            groupRefLabel: defectTypesLocalization[defectType.typeRef.toLowerCase()]
+              ? formatMessage(defectTypesLocalization[defectType.typeRef.toLowerCase()])
+              : defectType.longName,
             longName: defectType.longName,
             shortName: defectType.shortName,
```
Comment (AmsterGet) [inline app/src/pages/inside/common/utils.js:179]: Seems like the `shortName` used together with `groupRefLabel` every time.
Can we join these two properties into one?

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/pages/inside/common/utils.js (hunk 2 above)
Comment (tr1ble) [reply app/src/pages/inside/common/utils.js:179]: It used together here, but FE in QA we don't need `groupRefLabel`

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**